meson.build

```diff
@@ -14,12 +14,9 @@ host_system = host_machine.system()
 c_args = [
 	'-D_GNU_SOURCE',
 	'-fvisibility=hidden',
+	'-Wmissing-prototypes',
```

Comment:
It was kinda suggested by the Weston developers, see: https://gitlab.freedesktop.org/wayland/weston/-/merge_requests/362#note_394965

I made it an optional subproject, similar to how aml is optional to neatvnc. The idea is to avoid having to use version numbers and just use hashes this early, as things are probably still in flux.

Reply:
I was just wondering why making it a subproject requires changing `c_args`. More warnings are better though, so I'm not against this change. I might even end up setting `-Wall` or even `-Wextra` one day.